lib/FilesHooks.php

```diff
@@ -145,45 +153,57 @@ public function fileCreate($path) {
 		}
 
 		if ($this->currentUser->getUserIdentifier() !== '') {
-			$this->addNotificationsForFileAction($path, Files::TYPE_SHARE_CREATED, 'created_self', 'created_by');
+			$this->addNotificationsForFileAction($path, 'created_self', 'created_by');
 		} else {
-			$this->addNotificationsForFileAction($path, Files::TYPE_SHARE_CREATED, '', 'created_public');
+			$this->addNotificationsForFileAction($path, '', 'created_public');
 		}
 	}
 
 	/**
 	 * Store the update hook events
+	 *
 	 * @param string $path Path of the file that has been modified
 	 */
 	public function fileUpdate($path) {
-		$this->addNotificationsForFileAction($path, Files::TYPE_SHARE_CHANGED, 'changed_self', 'changed_by');
+		$this->addNotificationsForFileAction($path, 'changed_self', 'changed_by');
 	}
 
 	/**
 	 * Store the delete hook events
+	 *
 	 * @param string $path Path of the file that has been deleted
 	 */
 	public function fileDelete($path) {
-		$this->addNotificationsForFileAction($path, Files::TYPE_SHARE_DELETED, 'deleted_self', 'deleted_by');
+		$this->addNotificationsForFileAction($path, 'deleted_self', 'deleted_by');
```

Comment:
this breaks the non-monochrome activity icon handling across all clients, which is why I had pushed a revert on that in previous PRs.